modules/ROOT/pages/vpn-create-arm.adoc

```diff
@@ -0,0 +1,71 @@
+= Create an Anypoint VPN
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+== Prerequisites
+
+You must create a Virtual Private Cloud (VPC) before you can create a VPN. See xref:runtime-manager::virtual-private-cloud.adoc[Virtual Private Cloud] for more information. 
+
+You should also ensure that the components of your external network are configured correctly.
+
+When creating or configuring a VPN, you must provide the following information:
+
+* *Remote IP address:* IP address of the device you are connecting to.
+* *Pre-shared Key (PSK):* Shared secret used by the network tunnel that connects Anypoint Platform and your network. This value will be auto-generated if you select Automatic Tunnel Configuration.
+* *Point-to-Point CIDR:* IP range used by the network tunnel. You must configure a unique IP address range for each tunnel. This value will be auto-generated if you select Automatic Tunnel Configuration.
+* *Remote and Local Autonomous System Number (ASN)*: Specifies the collection of routing prefixes. You must configure both a remote and local ASN before creating a VPN. This is required only if you are using dynamic routing.
+* *Classless Inter-Domain Routing (CIDR)*: Specifies the routing address of your network. This is required only if you are using static routing.
+
+== Create a VPN
+
+. From the Anypoint Management Center, select *Runtime Manager*.
+. Go to the environment where you want to create a VPN.
+. Click the *VPNs* tab, then click *Create VPN*.
+. Enter or select the following information for your VPN: +
+* *Name* - Enter a name for your VPN. 
+* *VPC* - From the drop-down list, select the Virtual Private Cloud for the VPN connection.
+* *Remote IP Address* - Enter the remote IP address to use to connect to the VPN.
+. Select the routing type to use: 
++
+[NOTE]
+You can select either *BGP* or *Static*. The Border Gateway Protocol (BGP) type is recommended if it is supported by your device.
++
+.. To use BGP, select *BGP* and enter the following values: +
+* *Remote ASN* - Enter the ASN corresponding to your backend under Remote ASN. The default value is 65001.
```

Comment:
Change to:
Remote ASN - Enter the ASN corresponding to your backend. The default value is 65001.
Local ASN - Assign an ASN for the MuleSoft side. The default value is 64512.